index.js

```diff
@@ -70,6 +73,29 @@ async function run() {
       })
     }
 
+    if (logConfigurationLogDriver) {
+      if (!containerDef.logConfiguration) { containerDef.logConfiguration = {} }
+      const validDrivers = ["json-file", "syslog", "journald", "gelf", "fluentd", "awslogs", "splunk", "awsfirelens"];
+      if (!validDrivers.find(driver => driver === logConfigurationLogDriver)) {
```

Comment:
can this be simplified?
```
if (!validDrivers.includes(logConfigurationLogDriver)){
        throw new Error(`'${logConfigurationLogDriver}' is invalid logConfigurationLogDriver. valid options are ${validDrivers}. More details: https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_LogConfiguration.html`)
      }
```

Reply:
i could remove the link to the documentation or the valid options list. what do you think?

Reply:
what i mean by this [comment ](https://github.com/aws-actions/amazon-ecs-render-task-definition/pull/255/files#r1281174921)is that

instead of using array function `if (!validDrivers.find(driver => driver === logConfigurationLogDriver))`
can you just use `if (!validDrivers.includes(logConfigurationLogDriver))`?

Reply:
ahhh right away!